python/podio_class_generator.py

```diff
@@ -177,7 +177,9 @@ def _eval_template(self, template, data):
   def _write_file(self, name, content):
     """Write the content to file. Dispatch to the correct directory depending on
     whether it is a header or a .cc file."""
-    if name.endswith("h"):
+    if name.endswith("LinkDef.h"):
+      fullname = os.path.join(self.install_dir, "src", name)
+    elif name.endswith("h"):
```

Comment:
A very similar check is also need in `_write_cmake_lists_file` to avoid putting the `LinkDef.h` file into the list of generated headers that cmake then consumes.

Here: https://github.com/AIDASoft/podio/blob/60dcd12f062658cbd62442d83514b2eb9f140277/python/podio_class_generator.py#L441